rice-datanode/source/data_transfer_server.cc

```diff
@@ -70,11 +74,13 @@ void TransferServer::handle_connection(tcp::socket sock) {
 }
 
 void TransferServer::processWriteRequest(tcp::socket& sock) {
+	xmits++;
```

Comment:
Can the xmit operations all move to handle_connection?
Increment on calling write/read, decrement on return.